UIforETW/Utility.h

```diff
@@ -134,4 +140,31 @@ void SetCurrentThreadName(PCSTR threadName);
 
 void CopyStartupProfiles(const std::wstring& exeDir, bool force);
 
-void CloseValidHandle( _Pre_valid_ _Post_ptr_invalid_ HANDLE handle );
+void CloseFindHandle(_In_ _Pre_valid_ _Post_ptr_invalid_ HANDLE handle, PCWSTR directory);
+void CloseValidHandle(_In_ _Pre_valid_ _Post_ptr_invalid_ HANDLE handle);
+void CloseRegKey( _In_ _Pre_valid_ _Post_ptr_invalid_ HKEY key, PCWSTR const keyName );
+
+_Success_(return)
+bool OpenRegKey( _Out_ HKEY* key, _In_ HKEY root, PCWSTR const subkey );
+void ClipboardClose();
+bool ClipboardOpen();
+bool ClipboardEmpty();
+
+_Success_(return)
+bool ClipboardSetUnicode(_In_ const HGLOBAL hmem);
+
+_Success_(return)
+bool ClipboardGetUnicode(_Out_ _Post_valid_ HANDLE* const hClip);
+
+_Success_(return)
+bool AllocGlobalMemoryForString(const std::wstring& text, _Out_ _Post_valid_ HGLOBAL* const hmem);

```

Comment:
I'm actually not sure why I didn't put these in an anonymous namespace (in the `.cpp` file). Fixed.

`outputLastError` and `debugLastError` *might* be useful elsewhere, but I can totally remove them from the header if you'd like.